### DISCUSSION TITLE
5 Years Estimated to Win ?? Help

### DISCUSSION THREAD

**Discussion by Christopherkynn** [Chia-Network/chia-blockchain#2723] [Q&A]
Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz   1.80 GHz
8.00 GB (7.84 GB usable)8.00 GB (7.84 GB usable)
64-bit operating system, x64-based processor

1T External HD - Put Both Temp and Plot on the EHD. 

Would someone please guide me in the right direction to speed this up  ?

**Comment by mattuso83** [comment]
if you have around 1Tb of plots, right now you should see less than 1 year as estimated time to win a plot (I've around 500 gb of plots and I see 1 year).
If you want to see a lower amount of time, you just need to plot more blocks.

**Comment by OversizedWizard** [comment]
I have 28 plots at the moment. Telling me 2 months for reward.
I added 6 today and that didn't change. 


You need a crap load of hard drives.

Im gonna laugh my ass off when the price of the coin comes out and it's pennies and
